sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/SpecificParquetRecordReaderBase.java

```diff
@@ -28,12 +28,9 @@
 import java.util.Map;
 import java.util.Set;
 
-import com.google.common.annotations.VisibleForTesting;
-import org.apache.parquet.VersionParser;
-import org.apache.parquet.VersionParser.ParsedVersion;
-import org.apache.parquet.column.page.PageReadStore;
```

Comment:
Can we fix `dev/checkstyle.xml` to enforce this order?

Reply:
Ok, that's better, and once and for all. I'll make some changes later.

Reply:
After pr #35945 and discussion in pr #35970, the `ImportOrder` todo has been removed, the conclusion is that it's not worth changing all unordered files.
So shall we just make a little change for closing the related issued opened by uses?